nutkit/frontend/driver.py

```diff
@@ -128,16 +131,20 @@ def close(self):
 
     def session(self, access_mode, bookmarks=None, database=None,
                 fetch_size=None, impersonated_user=None,
-                ignore_bookmark_manager=None):
+                bookmark_manager: Optional[BookmarkManager] = None):
         req = protocol.NewSession(
             self._driver.id, access_mode, bookmarks=bookmarks,
             database=database, fetchSize=fetch_size,
             impersonatedUser=impersonated_user,
-            ignore_bookmark_manager=ignore_bookmark_manager
+            bookmark_manager=bookmark_manager
         )
         res = self.send_and_receive(req, allow_resolution=False)
         if not isinstance(res, protocol.Session):
             raise Exception("Should be session")
+
+        if bookmark_manager is not None:
+            self._bookmarks_managers[bookmark_manager.id] = bookmark_manager
```

Comment:
As `BookmarksSupplierRequest` might come during session creation, would it be possible to move this block to line 141 please?

Reply:
Sure.